edgexfoundry/edgex-compose
# refactor: Remove redundant Env Overrides

Removed the redundent override which are already defaults in code or entrypoint scripts







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>


## Testing Instructions
run `make pull ds-virtual delayed-start` to ensure latest images are used
run `make run ds-virtual delayed-start` 
Verify Spiffy Token Provider logs sure successfully generating token for Device Virtual and setting the `redisdb` secret in Device Virtual secret store
Verify Device Virtual logs show successful bootstrap and connection to MessageBus with Redis credentials. 
Verify Core Data finds its vault token and connects to Redis
